docs/source/03_tutorial/05_namespacing_pipelines.md

```diff
@@ -0,0 +1,229 @@
+
+    ```yaml
+
+    active_modelling_pipeline.regressor:
+      type: pickle.PickleDataSet
+      filepath: data/06_models/regressor_active.pickle
+      versioned: true
+      layer: models
+
+    candidate_modelling_pipeline.regressor:
+      type: pickle.PickleDataSet
+      filepath: data/06_models/regressor_candidate.pickle
+      versioned: true
+      layer: models
+    
+    ```
+
+3. Update the code in `pipelines/data_science/pipeline.py` using the snippet below.
+
+    <details>
+    <summary><b>Click to expand</b></summary>
+
+    ```python
+    from kedro.pipeline import Pipeline, node
+    from kedro.pipeline.modular_pipeline import pipeline
+
+    from .nodes import evaluate_model, split_data, train_model
+
+
+    def create_pipeline(**kwargs) -> Pipeline:
+        pipeline_instance = Pipeline(
+            [
+                node(
+                    func=split_data,
+                    inputs=["model_input_table", "params:model_options"],
+                    outputs=["X_train", "X_test", "y_train", "y_test"],
+                    name="split_data_node",
+                ),
+                node(
+                    func=train_model,
+                    inputs=["X_train", "y_train"],
+                    outputs="regressor",
+                    name="train_model_node",
+                ),
+                node(
+                    func=evaluate_model,
+                    inputs=["regressor", "X_test", "y_test"],
+                    outputs=None,
+                    name="evaluate_model_node",
+                ),
+            ]
+        )
+        ds_pipeline_1 = pipeline(
+            pipe=pipeline_instance,
+            inputs="model_input_table",
+            namespace="active_modelling_pipeline",
+        )
+        ds_pipeline_2 = pipeline(
+            pipe=pipeline_instance,
+            inputs="model_input_table",
+            namespace="candidate_modelling_pipeline",
+            parameters={"params:model_options": "params:model_options_experimental"},
+        )
+
+        return ds_pipeline_1 + ds_pipeline_2
+    ```
+
+    </details>
+
+### Let's explain what's going on here
+
+Modular pipelines allow you instantiate multiple instances of pipelines with static structure, but dynamic inputs/outputs/parameters.
+
+```python
+pipeline_instance = Pipeline(...)
+
+ds_pipeline_1 = pipeline(
+    pipe=pipeline_instance,
+    inputs="model_input_table",
+    namespace="active_modelling_pipeline",
+)
+ds_pipeline_2 = pipeline(
+    pipe=pipeline_instance,
+    inputs="model_input_table",
+    namespace="candidate_modelling_pipeline",
+    parameters={"params:model_options": "params:model_options_experimental"},
+)
+```
+
+> The `pipeline_instance` variable is our 'template' pipeline, `ds_pipeline_1` and `ds_pipeline_2` are our parametrised instantiations.
+
+The table below describes what the purpose of the keyword arguments in detail:
```

Comment:
```suggestion
The table below describes what the purpose is of the keyword arguments in detail:
```

Reply:
Sorry, this one is not gramatically correct after my addition... I think we should move `is` after `arguments`...

Reply:
You're right I've updated is as follows:
> The table below describes the purpose of each keyword arguments in detail